docs/dev/i18n.rst

```diff
@@ -84,5 +84,5 @@ A list of all languages supported by LuCI can be found in the ``packages/luci/lu
 Gluon's dependencies have been downloaded using ``make update``. Adding translations for these
 languages is straightforward using the ``msginit`` command.
 
-For other languages, support must be added tu LuCI first, which constitutes completely translating
-the ``base.pot``. Please contact the upstream LuCI maintainers if you'd like to do this.
+For other languages, support must be added to LuCI first, which constitutes completely translating
+the ``base.pot``. Please contact the upstream LuCI maintainers if you'd like to do this at https://github.com/openwrt/luci/ .
```

Comment:
The ordering is wrong, the sentence should be "Please contact the upstream LuCI maintainers at https://github.com/openwrt/luci/ if you'd like to do this"